android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDatabaseMigration.java

```diff
@@ -17,31 +18,18 @@
     @Override
     public void migrate(DynamicRealm dynamicRealm, long oldVersion, long newVersion) {
 
-        // Add schema increments and change methods when updating models
-
-        /* Example migration :
-
         // DynamicRealm exposes an editable schema
-        RealmSchema schema = realm.getSchema();
+        RealmSchema schema = dynamicRealm.getSchema();
```

Comment:
Realm schema version was not changed at the time of declaration. See the link I posted